src/main.rs

```diff
@@ -1,8 +1 @@
-mod cli;
-mod config;
-mod jsonrpc;
-mod lotus;
-mod manager;
-mod server;
-
-fn main() {}
+fn main() {}
```

Comment:
Maybe revert changes like these to avoid noise in the PR and in the git history.

Reply:
updated this to resolve the conflict, resolving